styles/colors-cobalt-day.less

```diff
@@ -86,7 +88,7 @@
 // ---------------------------------------
 @agate-checkbox-text-color: @agate-white;
 @agate-checkbox-bg-color: @agate-accent;
-@agate-checkbox-focus-bg-color: @agate-background-focus;
+@agate-checkbox-focus-bg-color: @agate-focus-bg-color;
```

Comment:
Is there a reason not to update @agate-picker-item-focus-bg-color like Checkbox?

Reply:
you are right. it can be updated